nntrainer/layers/embedding.cpp

```diff
@@ -61,9 +61,9 @@ void EmbeddingLayer::finalize(InitLayerContext &context) {
   dim.width(out_dim);
   dim.batch(1);
 
-  weight_idx = context.requestWeight(
-    dim, weight_initializer, weight_regularizer, weight_regularizer_constant,
-    context.getName() + ":Embedding", true);
+  weight_idx =
+    context.requestWeight(dim, weight_initializer, weight_regularizer,
+                          weight_regularizer_constant, "Embedding", true);
```

Comment:
not related to the PR it self but maybe change "Embedding-> "embedding"??

Reply:
same here.